performance-tests/commoncrawl-project/gradle.properties

```diff
@@ -0,0 +1,122 @@
+# These settings are used by Data Hub when
+# communicating with MarkLogic.
+# The values in this file are meant as project-wide settings.
+# You can override these properties for a specific environment
+# by creating a gradle-{environment}.properties file.
+# For example, to create a properties file for your prod environment create a file
+# named gradle-prod.properties.
+#
+# ....
+mlDHFVersion=5.3.0
+mlHost=localhost
+testing=true
+
+# If you are working with a load balancer please specify it here
```

Comment:
Remove everything else in this file except for the *Password properties at the end for flow-developer and flow-operator - it's all boilerplate that we don't need, DHF knows all these default values. 

mlUsername and mlPassword should then be defined in gradle-local.properties